vocab/credentials/v2/vocabulary.yml

```diff
@@ -17,160 +17,145 @@ ontology:
 
   - id: holder
     label: Holder
     domain: cred:VerifiablePresentation
+    defined_by: https://www.w3.org/TR/vc-data-model-2.0/#defn-holder
     range : IRI
-    comment: |
-      The value of the `holder` property MUST be a URI. It is RECOMMENDED that dereferencing the URI results in a document containing machine-readable information about the holder that can be used to verify the information expressed in the credential.
-
+    
   - id: issuanceDate
     label: Issuance date
-    domain: cred:VerifiableCredential
+    defined_by: https://www.w3.org/TR/2022/REC-vc-data-model-20220303/#issuance-date
     range: xsd:dateTime
     deprecated: true
-    comment: |
-      The value of the `issuanceDate` property was used to represents the earliest date when the information associated with the <a href="#credentialSubject">`credentialSubject`</a> property became valid. This property has been deprecated in favour of <a href="#validFrom">`validFrom`</a>.
-
-  - id: issued
-    label: issue date
-    domain: cred:VerifiableCredential
-    range: xsd:dateTime
-    comment: |
-      The value of the `issued` property MUST be a string value of an ISO8601 combined date and time string representing the date and time the credential was issued. Note that this date represents the earliest date when the information associated with the <a href="#credentialSubject>`credentialSubject`</a> property became valid.
+    comment: The value of the property was used to represent the earliest date when the information associated with the <a href="#credentialSubject">credentialSubject</a> property became valid. This property has been deprecated in favor of <a href="#validFrom">validFrom</a>.
 
   - id: issuer
-    label: issuer
+    label: Issuer
+    defined_by: https://www.w3.org/TR/vc-data-model-2.0/#defn-issuer
     domain: cred:VerifiableCredential
     range: IRI
-    comment: |
-      The value of the `issuer` property MUST be a URI. It is RECOMMENDED that dereferencing the URI results in a document containing machine-readable information about the issuer that can be used to verify the information expressed in the credential.
-
+    comment: The value of the property must be a URL uniquely identifying the issuer.
+    
   - id: refreshService
-    label: refresh service
+    label: Refresh service
+    defined_by: https://www.w3.org/TR/vc-data-model-2.0/#defn-refreshService
+    domain: cred:VerifiableCredential
+    status: reserved
+    range: cred:RefreshService
+
+  - id: renderMethod
+    label: Render method
     domain: cred:VerifiableCredential
+    defined_by: https://w3c-ccg.github.io/vc-render-method/#the-rendermethod-property
+    status: reserved
     range: cred:RefreshService
-    comment: The value of the `refreshService` property MUST be one or more Refresh Service instances such that the holder can refresh the credential.
 
   - id: serviceEndpoint
-    label: service endpoint
+    label: Service endpoint
```

Comment:
This seems like DID Core Terminology.

Reply:
Maybe... but how did this land in this vocabulary?

Reply:
I wonder whether this was added during the expansion of the issuer-holder-verifier trinity to include coordinator, service, etc., for each?